### ISSUE TITLE
LIP 1: Retroactive and Burn

### ISSUE THREAD

**Issue by IvanGBi** [lobster-dao/overview#13]
This Lobster Improvement Proposal is based on extensive Telegram, Twitter, and Github discussions from [previous issue](https://github.com/lobster-dao/overview/issues/10). This is just my imho, as I am not in control of the multisig or the minting or the deployment. If the DAO decides the same or differently, the execution would need to hit quorum. I rest my case.

There were two objectives which had to be addressed:

1. If there should be any drop to those who missed the bot, and if so - how.
2. If there should be burn, extra distribution, or other actions pre-reveal.

### Retroactive late airdrop is not possible, unfortunately!

If you read this, please make sure you follow the context from the link quoted in the first sentence. You need to have context to be able to argue. Basically, it's physically-technically impossible to bend the ruled in such a way as to allow members to get their drop as they could have without distorting the flow too much. 

The issue is that the reveal gets delayed (which goes against the collection rules - and that is not okay). And post-reveal, it would be extremely unfair to the lootbox design because people will see what they can claim, and then they will get fucked by MEV (see the bottom of the post). Overall, please read the arguments in the previous issue on why extending bot or doing post-reveal is just bad and unfair. Therefore, unfortunately, as much as I personally like the idea of having everyone in - there doesn't seem to be a fair & just way to have the late joiners in. Yet. Those members who missed might get their NFT on some new distribution cycle, without any expectations. Because if you were active before but missed - but you are smart - you can be active again and somehow get in the club! Simple, right? And also simple from the tech perspective.

_The objective was never to cap it by time, or exclude anyone. This is a retroactive airdrop, so everyone who missed should potentially get it hypothetically. People could be on a retreat or busy with their launches, it's not unusual. Or they could have been a bit mia. I don't agree with "oef if you were not online for 2 weeks screw you" - it might be factually correct, but this is a genuine community, so let's not argue this way._

**Proposal 1.1: no changes to be made.** It goes as is. No tech works extra required.

### No burn or changes to the DAO portion

I personally don't think the DAO should have 49% as it's against what NFTs have had today - but maybe this is new? That was not the objective to keep so many, but that was how devs made the contract. And those are the rules = remaining go to DAO. Not anyone's fault that happened. So, artist value and other comments aside - doing NOTHING is the correct way, because it prompts the least intervention from anyone inside or outside. And one more argument... to burn, DAO first needs to claim (if I understand contracts correctly?) That means spending 300K USD in ETH gas fees to be able to burn. That's not something feasible. So DAO can't burn (?)

What happens then? Nothing. DAO has a right to the pieces but does not mint them. Everyone knows what outstanding pieces are, but DAO can decide to NEVER mint them. Or never use them. The point is: let people decide later. These can be used for anything or nothing. In 1 year or 100 years or never. I have no control of this, you have no control of this.

**Proposal 1.2: no changes to be made.** DAO decides later or never. Nobody will touch the collection.

PS IMPORTANT: I am not in control of this anymore. I don't control the collection, the multisig, the snapshot page, or anything. There are multiple admins and owners. I rest my case here, I have my own project(s) I am busy with. No work, feedback, or anything should be expected from me personally - or from admins. Do whatever you want. Enjoy art and piss off <3 The objective of this LIP is to also prove that the original members made something and shipped it away, retaining no decision-making powers or control: anon devs deployed it, and it works as such. That's all.

### KEY MEV NOTICE

If you know how MEV works, and if you have NFT claims outstanding - you can technically frontrun DAO claims later on to get the NFT that you want, because on how tokenID<>ipfs path get matched. This is just a heads up that if you already claimed - you will just get your path, so never mind. But those trying to claim later post-reveal can either get super lucky or super unlucky. At the time of writing, about 80% of people claimed, so this is more of a notification to the DAO if the DAO decides to ever mint. The DAO needs to be careful of MEV if it ever comes across to the rare pieces mint block. 

_I rest my case. Have people decide what they want. I will just watch from the sidelines after creating LIP 1._

**Comment by tartakovsky** [comment]
It seems that we are currently voting for options that can't be done then. If burn is too expensive to execute, then there won't be a burn. DAO doesn't have 300k even if it wanted to spend them burning.

Does anyone see any options except doing nothing as Ivan proposes?

**Comment by kkkrackpot** [comment]
> It seems that we are currently voting for options that can't be done then. If burn is too expensive to execute, then there won't be a burn. DAO doesn't have 300k even if it wanted to spend them burning.
> 
> Does anyone see any options except doing nothing as Ivan proposes?

To mint it gradually, one by one, for years. :)

**Comment by tartakovsky** [comment]
Sure, but that's == doing nothing on LIP1. Future distributions will be voted for in the future.

**Comment by kkkrackpot** [comment]
Btw, someone's already voting 

https://snapshot.org/#/10b57e6da0.eth/proposal/QmNsfWz7iYBhU28so49oCjVRv2FiDmn1LWykh1cNsB7pMa

**Comment by tartakovsky** [comment]
Yeah, me included and it's cute, since no other option except No burn seems to be possible to execute 😄

**Comment by Datascience-Blockchain** [comment]
allow initial minters to mint one more NFT each?

**Comment by IvanGBi** [comment]
> Btw, someone's already voting
> 
> https://snapshot.org/#/10b57e6da0.eth/proposal/QmNsfWz7iYBhU28so49oCjVRv2FiDmn1LWykh1cNsB7pMa

Correct way for governance is to discuss something before making something vote-able. Create LIP2 (now) please on github and write out the positions more clearly. Or respond to LIP 1.

**Comment by ncerovac** [comment]
> Yeah, me included and it's cute, since no other option except `No burn` seems to be possible to execute 😄 Much governance.

yeah, a lot of us jumped to conclusions before thinking it out lol. anyhow, this will get better, gg everyone XD

**Comment by ekius** [comment]
I pro "No burn" or "Burn the rare only" and "reward projects", the NFT left in treasury will be reward for each project do airdrop or claimable mint for LOBs holder. A nice touch like AB did, they celebrate an event with a minted squiggle .

**Comment by kkkrackpot** [comment]
> > Btw, someone's already voting
> > https://snapshot.org/#/10b57e6da0.eth/proposal/QmNsfWz7iYBhU28so49oCjVRv2FiDmn1LWykh1cNsB7pMa
> 
> Correct way for governance is to discuss something before making something vote-able. Create LIP2 (now) please on github and write out the positions more clearly. Or respond to LIP 1.

It's not my proposal, no idea who launched it...

**Comment by defidummy** [comment]
Wait for royalty to kick-in
when enough, use that to burn
keep it simple and stupid blz

most holders are already too rich
they didn't even claim their nft
why would there be participation in governance

Distribution is done.
Let it pamp
1 lob > 1 punk

**Comment by saszer** [comment]
How about putting some of them in a timelock or a vesting (using team.finance? ) , where after the timelock it gets listed on opensea for a lower price(one by one over the years) or can just be made available to claim for some terms.

**Comment by ogreports** [comment]
If the intention wasn't to exclude people who unfortunately weren't active in the past few weeks, this is what has be done. It's one thing to say we didn't want to, but it's not changing how things have been done. Fortunately there's a way to allow the late one to claim. 

It might work through the bot. Is there a way to set it up so it can ask addresses from people who didn't send it yet and then ask them to send the required fee for minting + sending the NFT ? This seems a bit of work to me but not something impossible.

It would delay the reveal but I fail to see how that's a bigger issue than leaving people aside.

**Comment by IvanGBi** [comment]
I am not against it at all, just "This seems a bit of work to me but not something impossible." is nobody there to really orchestrate it all properly. Because again, this either takes too long (reveal delayed for too long = not cool) or it's done post-reveal (then see the point about MEV, where the arbitrage might just take anything juicy and cool!) 

What if I missed?
- Keep contributing, you can get it later. Good work should be rewarded.
Does DAO burn?
- No, is too costly to burn. So they are there to mint for DAO...
- Does it mean DAO wants to mint?
- It doesn't have to. It can, for fun, but not really needed.

It's the most sensible way forward? Everything will work as coded, and those who missed - unfortunately - can always get it later if DAO agrees to some new distributions. Which I think, personally, it should. Simple and quick. Imho.

If somebody has more radical plans: code it. Make proposal. Nobody will work "for you". It's not a project, there is no team, and despite us all wanting to retain the same distribution sheet - we physically are unable to put more time into it.

**Comment by IvanGBi** [comment]
Please stop confusing other people if you have no clue how governance works / snapshot. Not it can't and shouldn't. Smh

> > **Potential problems with having 49% of NFTs in the treasury**
> 
> Is it possible to blacklist the DAO from voting at all?
> 
> I'm kinda surprized that the DAO (as an entity) has the right to vote on its own proposals with its own treasury. A catch-22 or similar paradox...

**Comment by bitphish** [comment]
I somewhat fall under the made a mistake category.  I checked the eligible list and searched my username instead of the name I use for my profile.  So I thought I wasn't eligible and didn't do the bot even though I was.

I don't think anyone can assume just because they didn't do the bot that they didn't care.  There are a variety of factors on why one might have missed the drop.  

I think it would be a good gesture to allow the remaining eligible people claim their NFT(s), if possible as it would open more members to the DAO.  

Also wouldn't it benefit the artist as well to have the collection open to more people?

**Comment by IvanGBi** [comment]
Discussing in new LIP: https://github.com/lobster-dao/overview/issues/25
